Add description of debugging test unit (#20494)
dotnet/docs: docs/core/tutorials/testing-library-with-visual-studio-code.md

   The tests pass.

## Debug tests

> You can use the same process shown in [Tutorial: Debug a .NET Core console application using Visual Studio Code](debugging-with-visual-studio-code.md) to debug code using your unit test project. Instead of starting the ShowCase app project, open *StringLibraryTest/UnitTest1.cs*, and select **Run All Test** that between lines 7 and 8. If you can't find it, press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> to open the command palette and enter **Reload Window**.

Reviewer: ```suggestion
If you're using Visual Studio Code as your IDE, you can use the same process shown in [Debug a .NET Core console application using Visual Studio Code](debugging-with-visual-studio-code.md) to debug code using your unit test project. Instead of starting the *ShowCase* app project, open *StringLibraryTest/UnitTest1.cs*, and select **Run All Tests** between lines 7 and 8. If you're unable find it, select <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> to open the command palette and enter **Reload Window**.
```

Reviewer 2: ```suggestion
If you're using Visual Studio Code as your IDE, you can use the same process shown in [Debug a .NET Core console application using Visual Studio Code](debugging-with-visual-studio-code.md) to debug code using your unit test project. Instead of starting the *ShowCase* app project, open *StringLibraryTest/UnitTest1.cs*, and select **Run All Tests** that between lines 7 and 8. If you're unable to find it, press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> to open the command palette and enter **Reload Window**.
```
Thank you for reviewing.

But the following sentence is maybe correct. I double-check relative docs like .NET Core Tutorial series.

> press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>

> Visual Studio code starts the test project with the debugger attached. Execution will stop at any breakpoint you've added to the test project or the underlying library code.

Reviewer: ```suggestion
Visual Studio Code starts the test project with the debugger attached. Execution will stop at any breakpoint you've added to the test project or the underlying library code.
```

Reviewer 2: Thank you!
